MediaBrowser.MediaEncoding/Encoder/MediaEncoder.cs

```diff
@@ -1018,7 +1018,8 @@ public IEnumerable<string> GetPrimaryPlaylistVobFiles(string path, IIsoMount iso
 
         public bool CanExtractSubtitles(string codec)
         {
-            return false;
```

Comment:
That's true, but the way that the function is currently being used it's irrelevant as it just needs to know if a subtitle can't be converted. External text-based subs can be converted.

Reply:
I am sorry but a function that return only false or true, is useless and should be removed (if at least it would look for some property of some sort to activate or deactivate) . Unless they wanted to do some kind of implementation later on but ...

My 2 cents

Reply:
The function is defined in an interface, `ITranscoderSupport`.

Reply:
ha ok sorry :P ignore my comment, I didn't read the code